scripts/o3de/o3de/android_support.py

```diff
@@ -986,8 +986,24 @@ def validate_ninja(android_config) -> (Path, str):
 {dependencies}
     api 'androidx.core:core:1.1.0'
     implementation 'com.android.billingclient:billing:6.1.0'
+    {implementations}  
 }}
+
+{plugins}
+"""
+
+# CARBONATED -- begin : additional libs for messaging
+ADDITIONAL_IMPLEMENTATIONS = """
```

Comment:
Rename this to "ADDITIONAL_DEPENDENCIES" please

Reply:
Renamed